file,time,cmd_output,err_output,Uniform,Timeout
/home/gilles/GillesTestModels/toybox2.cnf,60.062,,,N/A,TRUE
/home/gilles/GillesTestModels/s38417_3_2.cnf,60.062,,,N/A,TRUE
/home/gilles/GillesTestModels/s38584_7_4.cnf,60.063,,,N/A,TRUE
/home/gilles/GillesTestModels/coreboot.cnf,60.059,,,N/A,TRUE
/home/gilles/GillesTestModels/s27_7_4.cnf,0.525,b'Experiment: 0     of     1\nconstantFactor:39   boundFactor: 14.541336623567457   logBoundFactor: 3.8620879811712894  \ntj: 1789   totalLoops: 2     beta: 0.8181818181818182 epsilon: 0.3       \nnumSolutions: 568   loThresh:82     hiThresh: 485   \n*** Progress *** i: 0 j: 2 ---- 0 / 3578\nUsing sampler: UniGen\ncmd:  ./samplers/unigen --samples=1 /home/gilles/GillesTestModels/s27_7_4.cnf /tmp/s27_7_4.txt > /dev/null 2>&1\nNumber of solutions returned by sampler: 1\nUsing sampler: KUS\ncmd:  python3  KUS.py --samples=1 --outputfile /tmp/s27_7_4.txt /home/gilles/GillesTestModels/s27_7_4.cnf > /dev/null 2>&1\nNumber of solutions returned by sampler: 1\n',"b'Traceback (most recent call last):\n  File ""barbarik.py"", line 1089, in <module>\n    ok, breakExperiment = exp.one_experiment(experiment, j, i, numExperiments, tj)\n  File ""barbarik.py"", line 976, in one_experiment\n    chainFormulaConf = chainFormulaSetup(sampleSol, unifSol, self.numSolutions)\n  File ""barbarik.py"", line 684, in chainFormulaSetup\n    assert len(sampleSol.split()) > len(countList), ""There are not enough samples to proceed, sampler failed ?"" \nAssertionError: There are not enough samples to proceed, sampler failed ?\n'",N/A,FALSE
/home/gilles/GillesTestModels/busybox-1.18.0.cnf,60.059,,,N/A,TRUE
/home/gilles/GillesTestModels/FM-3.6.1-refined.cnf,60.029,,,N/A,TRUE
/home/gilles/GillesTestModels/blasted_TR_ptb_2_linear.cnf,60.059,,,N/A,TRUE
/home/gilles/GillesTestModels/toybox.cnf,60.059,,,N/A,TRUE
/home/gilles/GillesTestModels/blasted_case110.cnf,60.061,,,N/A,TRUE
/home/gilles/GillesTestModels/s27_3_2.cnf,0.557,b'Experiment: 0     of     1\nconstantFactor:39   boundFactor: 14.541336623567457   logBoundFactor: 3.8620879811712894  \ntj: 1789   totalLoops: 2     beta: 0.8181818181818182 epsilon: 0.3       \nnumSolutions: 568   loThresh:82     hiThresh: 485   \n*** Progress *** i: 0 j: 2 ---- 0 / 3578\nUsing sampler: UniGen\ncmd:  ./samplers/unigen --samples=1 /home/gilles/GillesTestModels/s27_3_2.cnf /tmp/s27_3_2.txt > /dev/null 2>&1\nNumber of solutions returned by sampler: 1\nUsing sampler: KUS\ncmd:  python3  KUS.py --samples=1 --outputfile /tmp/s27_3_2.txt /home/gilles/GillesTestModels/s27_3_2.cnf > /dev/null 2>&1\nNumber of solutions returned by sampler: 1\n',"b'Traceback (most recent call last):\n  File ""barbarik.py"", line 1089, in <module>\n    ok, breakExperiment = exp.one_experiment(experiment, j, i, numExperiments, tj)\n  File ""barbarik.py"", line 976, in one_experiment\n    chainFormulaConf = chainFormulaSetup(sampleSol, unifSol, self.numSolutions)\n  File ""barbarik.py"", line 684, in chainFormulaSetup\n    assert len(sampleSol.split()) > len(countList), ""There are not enough samples to proceed, sampler failed ?"" \nAssertionError: There are not enough samples to proceed, sampler failed ?\n'",N/A,FALSE
/home/gilles/GillesTestModels/s713_15_7.cnf,60.065,,,N/A,TRUE
